### DISCUSSION TITLE
Should fpdf2 unfork?

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/fpdf2#906] [Ideas]
Currently, fpdf2 is showed like this:

![image](https://github.com/py-pdf/fpdf2/assets/1658117/c2d63bed-668e-4fad-95c3-b84b84a4b5b3)

and

![image](https://github.com/py-pdf/fpdf2/assets/1658117/a8ac986e-89cc-4c43-bb52-d29a06647a2f)

I'm also uncertain how it looks like for people forking from fpdf2 when they make a PR. Does the PR go to fpdf2 or fpdf?

As there is no chance that fpdf2 gets merged back into fpdf, I suggest to [unfork fpdf2](https://stackoverflow.com/a/66470086/562769). You can add to the documentation where fpdf2 orginated from. Directly on the start page, it's distracting. It looks like an "unofficial" version.

What do you think?

**Comment by gmischler** [comment]
I have been pondering the same question for a while. At first mainly because Github search was broken in forked repositories, which they seem to have fixed since then. But that note on top does also imply that fpdf2 is not actually the "real thing", even though it really is.

Similarly important would be to get a prominent notice into [reingart/pyfpdf](/reingart/pyfpdf) that development has moved over here. Is the conversation with Mariano mentioned in #752 about archiving and/or handing over the old repository still going on?  Maybe our previous home [PyFPDF](https://github.com/PyFPDF) could serve as the common archiving location for preserving all the history.
On pepy, the old [pepy:fpdf](https://www.pepy.tech/projects/fpdf) still has several times as many downloads as [pepy:fpdf2](https://www.pepy.tech/projects/fpdf2), despite all those years without updates. All those eyeballs would be really helpful to support the actively developed project.

**Comment by Lucas-C** [comment]
I agree with you that unforking would probably be a good idea.

I may not have the time to do it myself before a few days,
but feel free to go on with this idea if you want 😉

**Comment by gmischler** [comment]
> I may not have the time to do it myself before a few days,

If I understand the information on the linked pages correctly, all it takes is a message to github support and they will take care of it.

**Reply by Lucas-C** [reply]
Great, sounds simple!
